### PR TITLE
2.0: make DOTNET_FRAMEWORK a runtime image envvar and document runtime envvars

### PR DESCRIPTION
**Pull request by tmds**
These envvars are interesting for anyone doing their own .s2i scripts.

### CODE REVIEW

**Comment by aslicerh** [comment]
This code looks good to me, and the CI failure isn't your fault. We need to sync with the container release. I'll put that out now.

**Comment by aslicerh** [comment]
[cibuild]

File: 2.0/build/test/run [hunk 1]
```diff
@@ -14,8 +14,14 @@
 #
 # Example usage: $ sudo ./test/run
 
-IMAGE_NAME=${IMAGE_NAME:-dotnet/dotnet-20-rhel7}
-RUNTIME_IMAGE_NAME=${RUNTIME_IMAGE_NAME:-dotnet/dotnet-20-runtime-rhel7}
+if [ "$BUILD_CENTOS" = "true" ]; then
+  IMAGE_NAME=${IMAGE_NAME:-centos/dotnet-20-centos7}
+  RUNTIME_IMAGE_NAME=${RUNTIME_IMAGE_NAME:-dotnet/dotnet-20-runtime-rhel7}
+else
+  IMAGE_NAME=${IMAGE_NAME:-centos/dotnet-20-centos7}
```
Comment (jerboaa) [inline 2.0/build/test/run]: This doesn't look right. `dotnet/dotnet-20-rhel7` maybe?

File: 2.0/build/test/run [hunk 2]
```diff
@@ -14,8 +14,14 @@
 #
 # Example usage: $ sudo ./test/run
 
-IMAGE_NAME=${IMAGE_NAME:-dotnet/dotnet-20-rhel7}
-RUNTIME_IMAGE_NAME=${RUNTIME_IMAGE_NAME:-dotnet/dotnet-20-runtime-rhel7}
+if [ "$BUILD_CENTOS" = "true" ]; then
+  IMAGE_NAME=${IMAGE_NAME:-centos/dotnet-20-centos7}
+  RUNTIME_IMAGE_NAME=${RUNTIME_IMAGE_NAME:-dotnet/dotnet-20-runtime-rhel7}
```
Comment (jerboaa) [inline 2.0/build/test/run]: This doesn't seem right. Change it to: `centos/dotnet-20-runtime-centos7`

File: 2.0/runtime/test/run [hunk 3]
```diff
@@ -11,15 +11,22 @@
...
+  IMAGE_NAME=${IMAGE_NAME:-centos/dotnet-20-runtime-centos7}
+else
+  IMAGE_NAME=${IMAGE_NAME:-dotnet/dotnet-20-runtime-rhel7}
+fi
 OPENSHIFT_ONLY=${OPENSHIFT_ONLY:-false}
 
 test_dir="$(readlink -zf $(dirname "${BASH_SOURCE[0]}"))"
 source ${test_dir}/testcommon
 
+dotnet_version_serie="2.0"
```
Comment (jerboaa) [inline 2.0/runtime/test/run]: Typo: `dotnet_version_series`

**Comment by tmds** [comment]
> There seem to be some inconsistencies. Thoughts?

Thank you for catching these. I hadn't run the build test script stand-alone and changes to it were wrong.

**Pull request merged**